benchmark.js

```diff
@@ -466,9 +478,27 @@ var Benchmark = (function() {
         Native[fgImplKey] = function() {
           var now = performance.now();
           startup.stopTimer("fgStartupTime", now);
-          startup.stopTimer("startupTime", now);
+          startup.stopTimer("startupTime", now - storage.startFGDelay);
```

Comment:
This should only happen if the warm benchmark is enabled. Otherwise total startup time is 10s less than it should be, f.e.:

|       Test      | Baseline Mean |     Mean    |  +/-  |   %   |   P   |     Min     |     Max     | 
|:----------------|--------------:|------------:|------:|------:|------:|------------:|------------:|
| startupTime     |           n/a |     3,933ms |   n/a |   n/a |   n/a |     2,951ms |     4,991ms | 
| vmStartupTime   |           n/a |     3,135ms |   n/a |   n/a |   n/a |     3,000ms |     3,277ms | 
| bgStartupTime   |           n/a |       959ms |   n/a |   n/a |   n/a |       867ms |     1,073ms | 
| fgStartupTime   |           n/a |     9,839ms |   n/a |   n/a |   n/a |     8,889ms |    10,974ms | 
| *totalSize*     |         *n/a* |  *22,049kb* | *n/a* | *n/a* | *n/a* |  *20,998kb* |  *22,927kb* | 
| *domSize*       |         *n/a* |      *75kb* | *n/a* | *n/a* | *n/a* |      *74kb* |      *76kb* | 
| *styleSize*     |         *n/a* |     *282kb* | *n/a* | *n/a* | *n/a* |     *275kb* |     *289kb* | 
| *jsObjectsSize* |         *n/a* |  *15,578kb* | *n/a* | *n/a* | *n/a* |  *14,914kb* |  *15,883kb* | 
| *jsStringsSize* |         *n/a* |     *901kb* | *n/a* | *n/a* | *n/a* |     *891kb* |     *924kb* | 
| *jsOtherSize*   |         *n/a* |   *4,949kb* | *n/a* | *n/a* | *n/a* |   *4,005kb* |   *5,602kb* | 
| *otherSize*     |         *n/a* |     *263kb* | *n/a* | *n/a* | *n/a* |     *217kb* |     *333kb* | 
| *USS*           |         *n/a* |  *60,951kb* | *n/a* | *n/a* | *n/a* |  *58,876kb* |  *62,448kb* | 
| *peakRSS*       |         *n/a* | *191,076kb* | *n/a* | *n/a* | *n/a* | *191,076kb* | *191,076kb* |